src/e-pr.js

```diff
@@ -134,27 +145,23 @@ function pullRequestSource(source) {
 }
 
 program
+  .command('open', null, { isDefault: true })
   .description('Open a GitHub URL where you can PR your changes')
-  .option(
-    '-s, --source <source_branch>',
-    'Where the changes are coming from',
-    guessPRSource(current()),
-  )
-  .option(
-    '-t, --target <target_branch>',
-    'Where the changes are going to',
-    guessPRTarget(current()),
-  )
+  .option('-s, --source [source_branch]', 'Where the changes are coming from')
+  .option('-t, --target [target_branch]', 'Where the changes are going to')
   .option('-b, --backport <pull_request>', 'Pull request being backported')
   .action(async (options) => {
-    if (!options.source) {
+    const source = options.source || guessPRSource(current());
+    const target = options.target || guessPRTarget(current());
```

Comment:
This change loses the nice help text showing the default. Is there an underlying reason which necessitates this change?

Previous, for example:

```
Options:
  -s, --source <source_branch>   Where the changes are coming from (default: "roller/chromium/main")
  -t, --target <target_branch>   Where the changes are going to (default: "main")
  ```

Reply:
This seems to only work now when running `e pr open --help`, likely due to the way commander's `isDefault: true` behavior works.

Reply:
Gotcha. Unfortunate but OK.